internal/client/run_test.go

```diff
@@ -0,0 +1,19 @@
+package client_test
+
+import (
+	"testing"
+
+	"github.com/aquasecurity/trivy/internal"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_run(t *testing.T) {
+	//app := internal.NewApp("foo.bar.baz")
+	//err := app.Run([]string{"client", "--format=template", `--template="{{ range . }} {{ .Target }} {{ end }}"`, "alpine:3.10.2"})
+	//assert.NoError(t, err)
```

Comment:
cruft?

Reply:
ah yes this file is cruft. I removed it now.